Graphite/GraphiteMetric.cs

```diff
@@ -51,9 +51,9 @@ public override GraphiteMetric Read(ref Utf8JsonReader reader, Type typeToConver
             if (reader.TokenType != JsonTokenType.StartObject)
                 throw new JsonException($"Can not deserialize {nameof(GraphiteMetric)}");
 
-            var path = "";
-            var is_leaf = "";
-            var name = "";
+            var path = string.Empty;

```

Comment:
I would prefer `String` since you are accessing a property of the static class `String`